docs/kbs_attestation_protocol.md

```diff
@@ -246,14 +246,12 @@ in `alg`.
 The [`Attestation`](#attestation) and the [Token Resource](#token-resource)
 `tee-pubkey` field contains a public key from a HW-TEE generated asymmetric
 key pair.
-This field follows the [JSON Web Key](https://www.rfc-editor.org/rfc/rfc7517)
-format:
 
 ``` json
 {
-    "kty": "$key_type",
     "alg": "$key_algorithm",
-    "k": "public_key"
+    "k_mod": "$pubkey modulus",
+    "k_exp": "$pubkey exponent"
```

Comment:
Apologies for asking for another change, but could you also please fix https://github.com/confidential-containers/kbs/blob/main/docs/kbs.yaml#L327 accordingly?

Reply:
Oh, well spotted, thx. Also there is a typo in the above docs, it's serialized with a hypen (`k-mod`). I changed the openapi definitions to `TeePubKey` and adjusted the description, since it's no longer supposed to be a JWK I think. 

The n/e change should be easier to apply, since we know now what needs to be changed